lib/fluent/plugin/out_indicative.rb

```diff
@@ -35,6 +35,7 @@ class Fluent::Plugin::IndicativeOutput < Fluent::Plugin::Output
   config_param :event_name_key, :string
   config_param :event_time_key, :string
   config_param :event_unique_id_keys, :array, value_type: :string
+  config_param :event_filter_key, :string, default: nil
```

Comment:
so you want to do the default `false` in the python side? I'll note this on the ticket

Reply:
I don't think we need to default to false? Rather just send false for things that should be excluded and let everything else in still?